citest/json_contract/observation_verifier.py

```diff
@@ -446,122 +447,78 @@ def export_to_json_snapshot(self, snapshot, entity):
     super(ObservationVerifierBuilder, self).export_to_json_snapshot(
         snapshot, entity)
 
-  def EXPECT(self, verifier):
-    """Starts the verifier expression.
+  def EXPECT(self, pred):
+    """Starts the verification expression with a predicate.
 
-    This starts the first verifier clause. Additional verifiers can be added
+    This starts the first verifier clause. Additional predicates can be added
     later by using either AND or OR depending on how to combine them.
 
     Args:
-       verifier: [ObservationVerifier]
+       pred: [ObservationPredicate]
     Returns:
        self
     """
     if self.__current_builder_conjunction:
       raise ValueError(
-          'EXPECT (or append_verifier) was already used.'
-          ' Use OR or AND to add more verifiers.')
+          'expect (or "AND()" or "OR()") was already used.'
+          ' Use "OR()" or "AND()" to add more predicates.')
```

Comment:
This means there's already a conjunction started, right? Seems like the first parenthesis `(or "AND()" or "OR()")` is a bit confusing here. Maybe something like `Conjunction already starting using `expect`, use `OR()` or `AND()` to add more predicates.